docs/style.css

```diff
@@ -226,11 +286,9 @@ h2:hover a .anchor-hint {
     grid-column: 1 / span 2;
 }
 
-@media (max-width: 1024px) {
-    header, .headline, .details {
-        padding: 1rem;
-    }
+/* Media queries */
```

Comment:
Nitpick: dat is wat mij betreft de extra bytes en schermruimte niet waard, uit de backets/opmaak/code zelf is dit al wel duidelijk?

```suggestion
```